### PR TITLE
fixed bug at 411.

### PR DESCRIPTION
**Pull request by jpzhen**

### CODE REVIEW

File: calculator.js [hunk 1]
```diff
@@ -442,7 +442,7 @@ $( document ).ready(function() {
     //   - Change "theme name" to a catchy name (can be your name)
     //   - Change the colors blue and red with colors of your choice.
     //   - Uncomment the call to addOption (remove the `//`)
-    // addOption("theme name", "linear-gradient(blue, red)", "red", "red");
+    addOption("alango name", "linear-gradient(red, green)", "blue", "white");
```
Comment (etcadinfinitum) [inline calculator.js]: Consider removing "name" from theme name if you feel like it! 
```suggestion
    addOption("alango", "linear-gradient(red, green)", "blue", "white");
```

**Comment by arad98** [comment]
put #149 in the header to automatically close the bug when the code is merged, also what is your theme called?

**Review by etcadinfinitum — COMMENTED** [review]
Looks fine, will wait for someone else to approve.

Would be nice to see the PR description filled in more.

Pro Tip: under "Related Issues", you can [use GitHub keywords to automatically close issues](https://help.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords) if you like; this can be done in the PR description or any of the commits on the feature branch (title or body).

**Review by mikechavez — APPROVED** [review]
If Lizzy said it's cool, then it must be : )

**Pull request merged**